examples/deployments/GKE/vllm/vllm.yaml

```
@@ -0,0 +1,157 @@
+        limits:
+          cpu: "5"
+          memory: "10Gi"
+      extraPodSpec:
+        mainContainer:
+          image: nvcr.io/nvidia/ai-dynamo/vllm-runtime:0.4.0
+          workingDir: /workspace/components/backends/vllm
+          command:
+            - /bin/sh
+            - -c
+          args:
+          - |
```

YAMLlint (1.37.1)

[warning] wrong indentation: expected 12 but found 10

(indentation)